packages/quantic/force-app/examples/main/lwc/exampleQuanticFacet/exampleQuanticFacet.js

```diff
@@ -19,6 +19,24 @@ export default class ExampleQuanticFacet extends LightningElement {
             label: 'Label',
             description: 'The label to use as the facet title.',
             defaultValue: 'Type'
+        },
+        {
+            attribute: 'numberOfValues',
+            label: 'Number of values',
+            description: 'The number of values the facet should display.',
```

Comment:
@acarpentier01 Can you please revise the description strings in this file? Thanks